src/renderer/components/ft-share-button/ft-share-button.js

```diff
@@ -75,10 +86,17 @@ export default defineComponent({
       return `https://www.youtube.com/channel/${this.id}`
     },
 
+    youtubePlaylistUrl() {
+      return `https://youtube.com/playlist?list=${this.id}`
+    },
+
```

Comment:
YouTube is _supposed_ to allow embed playlists as well:
`https://www.youtube.com/embed/videoseries?list=${this.id}`

![image](https://user-images.githubusercontent.com/78101139/215136834-70898843-a213-4832-9f73-551df55d6ac2.png)

Reply:
Okay it looks like it only works for some playlist but others spit out "Video unavailable".

Reply:
Added support for both YouTube and Invidious playlist embeds.

Reply:
I cant find a single playlist where it works for yohtube but FreeTube and Invidious both support those types of links. So, I'm fine without including it

Ex: url https://yt.artemislena.eu/embed/2fDzCWNS3ig?list=PLy_wKxVmWb4bayO1qxAENx_MkbxiUxODg

Reply:
Playlists on the LTT channel worked for me. e.g. https://www.youtube.com/playlist?list=PL8mG-RkN2uTwNy21Qm1qyCxoSTipfzJ7s => https://www.youtube-nocookie.com/embed/videoseries?list=PL8mG-RkN2uTwNy21Qm1qyCxoSTipfzJ7s